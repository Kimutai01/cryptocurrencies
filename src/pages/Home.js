import React from 'react';
import 'bootstrap/dist/js/bootstrap';
import 'bootstrap/dist/css/bootstrap.min.css';
import { Carousel } from 'react-bootstrap';
import image from '../assets/images/pink1.jpg';
import rates2 from '../assets/images/pink2.jpg';
import rates3 from '../assets/images/pink3.jpg';

const Home = () => (
  <div style={{ overflow: 'none' }}>
    <Carousel fade>
      <Carousel.Item>
        <img
          src={image}
          className="d-block w-100"
          alt="stock"
          style={{ height: '100vh' }}
        />
        <Carousel.Caption>
          <h3>Check accurate coin data ðŸ’¯</h3>
          <p>We provide accurate coin data all over the world.</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          src={rates2}
          alt="stock2"
          className="d-block w-100"
          style={{ height: '100vh' }}
        />
        <Carousel.Caption>
          <h3>Check data of all coins ðŸ˜‡</h3>
          <p>We provide data about all coins</p>
        </Carousel.Caption>
      </Carousel.Item>
      <Carousel.Item>
        <img
          src={rates3}
          alt="stock4"
          className="d-block w-100"
          style={{ height: '100vh' }}
        />
        <Carousel.Caption>
          <h3>We record changes every minuteðŸ‘Œ</h3>
          <p>We provide all crypto changes every minute</p>
        </Carousel.Caption>
      </Carousel.Item>
    </Carousel>
  </div>
);

export default Home;
