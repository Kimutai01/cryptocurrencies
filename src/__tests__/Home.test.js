import renderer from 'react-test-renderer';
import Home from '../pages/Home';

test('Home component testing', () => {
  const TREE = renderer.create(<Home />).toJSON();
  expect(TREE).toMatchSnapshot();
});
