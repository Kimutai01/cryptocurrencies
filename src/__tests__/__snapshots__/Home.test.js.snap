// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component testing 1`] = `
<div
  style={
    Object {
      "overflow": "none",
    }
  }
>
  <div
    className="carousel slide carousel-fade"
    onKeyDown={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
  >
    <div
      className="carousel-indicators"
    >
      <button
        aria-current={true}
        aria-label="Slide 1"
        className="active"
        data-bs-target=""
        onClick={[Function]}
        type="button"
      />
      <button
        aria-current={false}
        aria-label="Slide 2"
        data-bs-target=""
        onClick={[Function]}
        type="button"
      />
      <button
        aria-current={false}
        aria-label="Slide 3"
        data-bs-target=""
        onClick={[Function]}
        type="button"
      />
    </div>
    <div
      className="carousel-inner"
    >
      <div
        className="active carousel-item"
      >
        <img
          alt="stock"
          className="d-block w-100"
          src="coin.jpg"
          style={
            Object {
              "height": "100vh",
            }
          }
        />
        <div
          className="carousel-caption"
        >
          <h3>
            Check accurate coin data ðŸ’¯
          </h3>
          <p>
            We provide accurate coin data all over the world.
          </p>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="stock2"
          className="d-block w-100"
          src="tether.jpg"
          style={
            Object {
              "height": "100vh",
            }
          }
        />
        <div
          className="carousel-caption"
        >
          <h3>
            Check data of all coins ðŸ˜‡
          </h3>
          <p>
            We provide data about all coins
          </p>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="stock4"
          className="d-block w-100"
          src="main.jpg"
          style={
            Object {
              "height": "100vh",
            }
          }
        />
        <div
          className="carousel-caption"
        >
          <h3>
            We record changes every minuteðŸ‘Œ
          </h3>
          <p>
            We provide all crypto changes every minute
          </p>
        </div>
      </div>
    </div>
    <a
      className="carousel-control-prev"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
      <span
        className="visually-hidden"
      >
        Previous
      </span>
    </a>
    <a
      className="carousel-control-next"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex={0}
    >
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
      <span
        className="visually-hidden"
      >
        Next
      </span>
    </a>
  </div>
</div>
`;
